tests/test_autotuner.py

```diff
@@ -101,7 +101,7 @@ def test_timesteps_per_at_run():
         stencil = Eq(outfield.indexed[t + to, x, y, z],
                      outfield.indexify() + infield.indexify()*3.0)
         op = Operator(stencil, dle=('blocking', {'blockalways': True}))
-        op(infield=infield, outfield=outfield, autotune=True)
+        op(infield=infield, outfield=outfield, t=2, autotune=True)
```

Comment:
?

Reply:
Ahhh, this is a subtle one. In the previous scheme, a stepping dimension would provide its buffer size as the default runtime loop bound. Since this makes no sense from a modellers perspective, this rule got dropped, meaning every time-dependente operator now needs to either use `save=some_value` or define the number of timesteps on invocation. This test does not (because it doesn't care), but this now gets picked up as a missing value for `time`, hence we need to provide one.

Reply:
OK

Reply:
> Ahhh, this is a subtle one. [...] hence we need to provide one.

I had to apply the same fix to another couple of tests